profiles/base.yaml

```diff
@@ -215,6 +215,8 @@ symbiosis_learning_threshold: 2
 compost_room: 991
 mine_for_outdoorsmanship: false
 train_appraisal_with_gear: false
+train_appraisal_with_pouches: false
+full_pouch_container:
```

Comment:
Forward looking thought, we could have the places (lockpick and combat-trainer) that cycle pouches not use stow on full pouches and instead use this variable when available.